src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,30 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={selectedTab === tab ? 'is-active' : ''}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => {
+                if (selectedTab !== tab) {
```

Comment:
```suggestion
                if (selectedTab.id !== tab.id) {
```
better to check the id of the tab instead of the whole object